packages/components/ng-ovh-contracts/src/summary/template.html

```diff
@@ -18,6 +18,9 @@
             data-ng-href="{{ contract.url }}"
             rel="noopener"
             target="_blank"
+            data-track-on="click"
+            data-track-type="action"
+            data-track-name="{{ 'PublicCloud::pci::projects::onboarding::contract::' + contract.name }}"
```

Comment:
**nitpick:** This is a generic component used in multiple places. You can not add tracking specific to PCI here

Reply:
I've added a fix in this commit : https://github.com/ovh/manager/pull/10759/commits/a82892e9e733ef6e8ecdc3da0282599b5d6bdc9c